src/config.ts

```diff
@@ -65,6 +65,10 @@ class Config {
 		return this.workspaceConfiguration.get('tabIconColourTheme', 'colour');
 	}
 
+	public retainContextWhenHidden(): boolean {

```

Comment:
Can this method please be moved to line 52so that the config methods are kept in alphabetical order.